Leanplum-tvOS-SDK.podspec

```diff
@@ -0,0 +1,22 @@
+Pod::Spec.new do |s|
+  s.name = 'Leanplum-tvOS-SDK'
+  s.version = '2.0.2'
+  s.summary = 'Mobile Marketing Platform. Integrated. ROI Engine.'
+  s.description = 'Leanplum’s integrated solution delivers meaningful engagement across messaging and the in-app experience. We offer Messaging, Automation, App Editing, Personalization, A/B Testing, and Analytics.'
+  s.homepage = 'https://www.leanplum.com'
+  s.license = { :type => 'Commercial', :text => 'See https://www.leanplum.com/tos' }
+  s.author = { 'Leanplum' => 'support@leanplum.com' }
+  s.social_media_url = 'https://twitter.com/leanplum'
+  s.tvos.deployment_target = '8.0'
+  s.requires_arc = true
+  s.source = { :git => 'https://github.com/Leanplum/Leanplum-iOS-SDK.git', :tag => s.version.to_s }
```

Comment:
https://github.com/Leanplum/Leanplum-tvOS-SDK

Reply:
there is no https://github.com/Leanplum/Leanplum-tvOS-SDK, both podspecs share same github.

Reply:
oh, are you trying to attach the release in the same repository?
I was thinking it will be similar to LocationAndBeacons. Have a repository that pretty much only have github releases. https://github.com/Leanplum/Leanplum-iOS-LocationAndBeacons

Having two binaries on github is not bad of an idea too. It's just different... Actually doesn't this apply to all of them? For ex. iOS release can have both dynamic and static. 
Location release can have Location, Beacons, and static versions of each.

Reply:
yeah actually you are right, its better to have it in separate repo.